addons/versioning/XEH_postInitServer.sqf

```diff
@@ -18,23 +18,53 @@ FUNC(paranoid) = {
     };
 };
 
-QGVAR(versions_serv) addPublicVariableEventHandler { (_this select 1) call FUNC(paranoid) };
+QGVAR(versions_serv) addPublicVariableEventHandler {(_this select 1) call FUNC(paranoid)};
 
 // Missing Modfolder check
 FUNC(handleMismatch) = {
     params ["_machine","_mod"];
-    [format["%1 - Not running! (Machine: %2)", _mod, _machine], QUOTE(COMPONENT), [CBA_display_ingame_warnings, true, true]] call CBA_fnc_debug;
+    [format ["%1 - Not running! (Machine: %2)", _mod, _machine], QUOTE(COMPONENT), [CBA_display_ingame_warnings, true, true]] call CBA_fnc_debug;
 };
 
 QGVAR(mismatch) addPublicVariableEventHandler { (_this select 1) call FUNC(handleMismatch) };
 
 private "_str";
-_str = 'if(isServer)exitWith{};if (isNil "CBA_display_ingame_warnings") then { CBA_display_ingame_warnings = true; };LOG("cba_versioning_check");0 = objNull spawn { sleep 1; sleep 1; _func={GVAR(mismatch)=[format["%2 (%1)",name player, player],_this];publicVariable QGVAR(mismatch);_this spawn{_t=format["You are missing the following mod: %1",_this];diag_log text _t;sleep 2;if (CBA_display_ingame_warnings) then {player globalChat _t}}};';
+
+_str = 'if (isServer) exitWith{};\
```

Comment:
This string was compacted for network performance reasons. Lack of proper formatting is intentional (and could be improved by removing even more spaces, actually)